lib/main.rb

```diff
@@ -1,11 +1,49 @@
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative 'app'
 
-person = Person.new(22, 'maximiliano')
+def print_menu
+  puts
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+  puts 'Enter your choice:'
+end
 
-capitalized_person = CapitalizeDecorator.new(person)
-puts "Capitalized name: #{capitalized_person.correct_name}"
+def handle_choice(choice, app)
+  case choice
+  when 1
+    app.list_books
+  when 2
+    app.list_people
+  when 3
+    app.create_person
+  when 4
+    app.create_book
+  when 5
+    app.create_rental
+  when 6
+    app.list_rentals_for_person
```

Comment:
- [ ] Please make sure to check the names of your methods! this `list_rentals_for_person` should be `list_rentals` according to the method in your `app.rb`. 🤓 🔧